python/lsst/meas/algorithms/psfDeterminer.py

```diff
@@ -19,25 +19,56 @@
 # the GNU General Public License along with this program.  If not,
 # see <http://www.lsstcorp.org/LegalNotices/>.
 #
-from lsst.pex.config import makeRegistry
-from .pcaPsfDeterminer import PcaPsfDeterminer
+from __future__ import absolute_import, division, print_function
 
-__all__ = ["psfDeterminerRegistry"]
+import abc
 
-psfDeterminerRegistry = makeRegistry(
-    '''A registry of PSF determiner factories
+import lsst.pipe.base as pipeBase
+import lsst.pex.config as pexConfig
 
-    A PSF determiner factory makes a class with the following API:
+__all__ = ["BasePsfDeterminerConfig", "BasePsfDeterminerTask", "psfDeterminerRegistry"]
 
-    def __init__(self, config, schema=None):
+class BasePsfDeterminerConfig(pexConfig.Config):
+    kernelSize = pexConfig.Field(
```

Comment:
I think I understand why you put these three settings in the base config, but is there some way to document/explain it? For example, I could see a justification for also including "borderWidth" in the base, although I realize that it's only not there because it's not used by PsfexPsfDeterminer. Is that correct?